### DISCUSSION TITLE
Is there any document that compares different model architectures?

### DISCUSSION THREAD

**Discussion by vahid9** [Anjok07/ultimatevocalremovergui#1418] [Q&A]
Hi, 
Is there any link that provides detailed structure/architecture of each neural model (the number of layers, type of layers, etc) that are available for UVR5?

**Comment by ethan1785ra** [comment]
this google docs will be useful to your question: https://docs.google.com/document/d/17fjNvJzj8ZGSer7c7OFe_CNfUKbAxEh_OBv94ZdRG5c/edit#heading=h.n8ac32fhltgg

**Reply by ACEXAN** [reply]
Thank you so much, this is very useful!
